runtime/compiler/optimizer/TreeLowering.cpp

```diff
@@ -459,6 +459,58 @@ TR::TreeLowering::fastpathAcmpHelper(TR::Node * const node, TR::TreeTop * const
       traceMsg(comp, "Call node n%dn is now in block_%d (prevBlock is %d)\n", node->getGlobalIndex(), callBlock->getNumber(), prevBlock->getNumber());
    prevBlock->append(TR::TreeTop::create(comp, checkRhsNull));
    cfg->addEdge(prevBlock, targetBlock);
+
+   if (!performTransformation(comp, "%sInserting fastpath for lhs is VT\n", optDetailString()))
+      return;
+
+   auto* const vftSymRef = comp->getSymRefTab()->findOrCreateVftSymbolRef();
+   auto* const classFlagsSymRef = comp->getSymRefTab()->findOrCreateClassFlagsSymbolRef();
+   auto* const j9ClassIsVTFlag = TR::Node::iconst(node, J9ClassIsValueType);
```

Comment:
@hzongaro has a commit : 

https://github.com/eclipse/openj9/pull/12126/commits/5262d42432c000d175ae57b888c89703d338e6c5

(that is being reviewed presently) that created a helper for this kind of logic in VMJ9.cpp

Reply:
FYI.  That commit has been squashed down into https://github.com/eclipse/openj9/pull/12126/commits/c9d6801068f9fd6a7df84c829d0bb4a2b5c692da (pull request #12126)

Reply:
I'll wait for #12126 to get merged so that I can rebase and pick up the new helper. The PR has been approved already so hopefully it won't be too long before it gets merged.

Reply:
FYI, Leo @Leonardo2718 - #12126 has been merged.

Reply:
Fixed in force push.